running-a-nats-service/configuration/leafnodes/leafnode_conf.md

```diff
@@ -96,6 +96,21 @@ If other form of credentials are used (jwt, nkey or other), then the server will
 | `ws_compression` | If connecting with [Websocket](leafnode_conf.md#connecting-using-websocket-protocol) protocol, this boolean (`true` or `false`) indicates to the remote server that it wishes to use compression. The default is `false`.                                                 |
 | `ws_no_masking`  | If connecting with [Websocket](leafnode_conf.md#connecting-using-websocket-protocol) protocol, this boolean indicates to the remote server that it wishes not to mask outbound WebSocket frames. The default is `false`, which means that outbound frames will be masked. |
 
+### Signature Handler
+
+As of NATS Server v.2.9.0, for users embedding the NATS Server, it is possible to replace the use of the credentials file by a signature callback which will sign the `nonce` and provide the JWT in the `CONNECT` protocol. The `RemoteLeafOpts` has a new filed:
```

Comment:
At the end of the sentence, should 'filed' be 'field'?

Reply:
Yes it should :-)